#175756592 Created a welcome Page for Visit Africa App

Floride125-art opened the pull request:
#### What does this PR do?
It indicates a welcome page task that I have worked on and some modification including arranging files in specific folders according to the folder structure so as to make it look great.
#### Description of Task to be completed?
The welcome Page task that I worked on is the first page a user see when open the visit Africa app. This page simply summarize what the visit Africa app do and allow user to continue to the next page where she or he can be able to register and create an account with  us. 
#### How should this be manually tested?
clone it  
run flutter pub get to install dependencies  
run flutter run to start
#### Any background context you want to provide?
As the user you should click on continue button to be able to proceed on the next pages of Visit Africa App.
#### What are the relevant pivotal tracker stories?
#175756592
#### Screenshots (if appropriate)
![welcomeIMg](https://user-images.githubusercontent.com/71024941/99674416-ace70800-2a7e-11eb-884a-2abefa1c097f.jpg)

#### Questions:
N/A

VedaDom commented on lib/welcomePage.dart:
```diff
@@ -0,0 +1,80 @@
+        child: Stack(
+          children: <Widget>[
+            Image.asset(
+              "assets/cover.png",
```
Can you put images in the images folder? `assets/images/example.png`

dianahub250 commented:
Great work! You can proceed!

Sandrine21 reviewed (approved):
The work is well done. I am good with it.

VedaDom commented on lib/signup.dart:
```diff
@@ -0,0 +1,15 @@
+      appBar: AppBar(
+        title: Text('Visit Africa'),
+        centerTitle: true,
+        backgroundColor: Color(0xffB15C1E),
```
Why do we have this `app bar ` here not in the original design? It kills everything. what do you think about removing it?

VedaDom commented on lib/welcomePage.dart:
```diff
@@ -0,0 +1,80 @@
+            Padding(
+              padding: const EdgeInsets.only(top: 470.0, left: 30.0),
+              child: Text(
+                'Welcome',
```
Aren't we supposed to have Welcome to **visit Africa** ? what changed your mind?

VedaDom commented on lib/welcomePage.dart:
```diff
@@ -0,0 +1,80 @@
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 630.0, left: 30.0),
```
Can you find another way to move this to bottom?

dianahub250 reviewed (approved):
This is the Good work!

mujohn26 reviewed (commented):
LGTM

Floride125-art committed:
feature(folder structure):modify folder structure
have to arrange well files in a folder structure very well
[finishes#175756592]
